cmd/pipecd/ops.go

```diff
@@ -98,6 +99,10 @@ func (s *ops) run(ctx context.Context, t cli.Telemetry) error {
 		}
 	}()
 
+	// Starting orphan commands cleaner
+	cleaner := orphancommandcleaner.NewOrphanCommandCleaner(ds, t.Logger)
+	cleaner.Run(ctx)
```

Comment:
This should be wrapped by `errgroup` to terminate `ops` when `cleaner` exit because of an error.

``` golang
group.Go(func() error {
	return cleaner.Run(ctx)
})
```

Reply:
IMO, there was no need to stop ops because errors that occur in cleaner is probably caused by a network error when communicating with datastore.

Of course, it is necessary to output the error log to show that an error has occurred. However, I don't think it is necessary to stop ops because updateOrphanCommandsStatus will include the update that should have been done with the previous process if the previous process failed.

Or how about stopping `ops` if it fails 3 times in a row?

What do you think? 🤔

Reply:
Nice comment. I got your points.
Agree with you that the `cleaner` may have some errors while executing its jobs.

But that is the problem of `cleaner`, not this `run` function.
This `run` should not care about what each component is doing, it just cares that they are fine or not. If any of them had a problem, `ops` should be stopped to notify that there was a problem from one of its components.
In other words, `run` should see all of its components (`cleaner`, `http-server`, `insightcollector`...) in the same way. 

About the network problem or something that can cause the error while `cleaner` executing the job,
`cleaner` can ignore them,  just log the error message, and does not return. 
By that way, the `ops` will not be stopped by those errors and we can keep `run` to be nice too.

Reply:
> About the network problem or something that can cause the error while cleaner executing the job,
cleaner can ignore them, just log the error message, and does not return.
By that way, the ops will not be stopped by those errors and we can keep run to be nice too.

OK, I see 👍